docs/nmodl/python_scripts/utils/cell.py

```diff
@@ -18,7 +18,12 @@ def create_circuit(self):
     def record(self):
         pass
 
-    def simulate(self, tstop, dt):
+    def simulate(self, tstop, dt, corenrn=False, gpu=False):
+        from neuron import coreneuron
+
+        coreneuron.enable = corenrn
+        coreneuron.gpu = gpu
+
         h.dt = dt
         h.finitialize()
         while h.t < tstop - dt / 2:
```

Comment:
@iomaganaris I didn't modify it because

> E       AttributeError: 'hoc.HocObject' object has no attribute 'stdinit'

Reply:
I think lines 28-30 should change to be:
```
h.cvode.cache_efficient(1)
pc = h.ParallelContext()
h.stdinit()
pc.psolve(tstop)
```

Reply:
`h.CVode().cache_efficient(1)`

But still the same error about `stdinit` ;)

Reply:
Ah yes.. When we do `from neuron import h` in line 1 we also need to import `gui`:
```
from neuron import h, gui
```